SU2_CFD/src/fluid/CMutationTCLib.cpp

```diff
@@ -59,8 +59,12 @@ CMutationTCLib::CMutationTCLib(const CConfig* config, unsigned short val_nDim):
 
   for(iSpecies = 0; iSpecies < nSpecies; iSpecies++) MolarMass[iSpecies] = 1000* mix->speciesMw(iSpecies); // x1000 to have Molar Mass in kg/kmol
 
-  if (mix->hasElectrons()) { nHeavy = nSpecies-1; nEl = 1; }
-  else                     { nHeavy = nSpecies;   nEl = 0; }
+  if (mix->hasElectrons()) { 
+    if(config->GetViscous()) {
+      SU2_MPI::Error(string("Ionization is not yet operational for a viscous flow in the NEMO solver."),
+      CURRENT_FUNCTION);}
+    else { nHeavy = nSpecies-1; nEl = 1;}}
```

Comment:
Stop coming up with new formatting styles please...
```suggestion
  if (mix->hasElectrons()) { 
    if (config->GetViscous()) {
      SU2_MPI::Error("Ionization is not yet operational for a viscous flow in the NEMO solver.", CURRENT_FUNCTION);
    } else {
      nHeavy = nSpecies-1;
      nEl = 1;
    }
  }
```